Description: Several fixes to make dma build on Debian.
 - use mtime instead of mtimespec.
Origin: other: http://svn.ringlet.net/svn/ringlet/mail/dma/
Forwarded: not-needed
Author: Peter Pentchev <roam@ringlet.net>
Last-Update: 2010-06-18

--- a/dma.c
+++ b/dma.c
@@ -295,7 +295,7 @@
 			exit(1);
 		}
 		if (gettimeofday(&now, NULL) == 0 &&
-		    (now.tv_sec - st.st_mtim.tv_sec > MAX_TIMEOUT)) {
+		    (now.tv_sec - st.st_mtime > MAX_TIMEOUT)) {
 			asprintf(__DECONST(void *, &errmsg),
 				 "Could not deliver for the last %d seconds. Giving up.",
 				 MAX_TIMEOUT);
